install/image-references

```diff
@@ -58,3 +58,7 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-kube-rbac-proxy:4.2.0
+  - name: ovirt-machine-controllers
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-ovirt-machine-controllers
```

Comment:
Same question as in the ConfigMap: I've never seen the tag omitted here. Is the tooling okay with that?

Reply:
https://github.com/openshift/machine-api-operator/pull/432#discussion_r368535776